_en/2015-08-11-about.html

```diff
@@ -14,24 +14,27 @@
             Vi har avdelinger over hele landet – Stavanger, Bergen, Trondheim, Vestfold, Sørlandet og Oslo. Hver eneste måned skjer
             det kule ting i javaBin-regi – både meetups, studentarrangementer, arrangementer for barn, og mye annet spennende!-->
 
-            javaBin is one of the largest community groups in Norway. It is driven by volunteer enthusiasts from across the country.
-            We have branches throughout the country - Stavanger, Bergen, Trondheim, Vestfold, Southern Norway and Oslo. We organize events
-            almost every month in all javaBin branches - both meetups, student events, events for kids, and much more exciting stuff!
+            javaBin is one of the largest communities in Norway. It is driven by volunteer enthusiasts from across the country. 
+            We have branches throughout the country – in Stavanger, Bergen, Trondheim, Vestfold, Southern Norway and Oslo. 
+            We organize events almost every month in all branches of javaBin. It is both meetups, events for students, 
```

Comment:
... in all branches of javaBin, including meeting, events for students, events for kids, and other exciting stuff!"